frontend/src/lib/notify/notify.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+  import { notifications, removeAllNotifications, removeNotification} from './index.js';
+</script>
+
+<div class="toast toast-center w-96">
+{#if $notifications.length > 1}
+<div class="mt-2">
```

Comment:
Our close buttons are always on the right, so that would be nice here too. I'm not quite sure if this will do it, but something like this. `mt-2` also looks like an oversight.
```suggestion
<div class="flex items-end">
```